frameworks/elastic/src/main/dist/svc.yml

```diff
@@ -281,6 +281,49 @@ pods:
           - name: node
             type: TLS
         {{/TASKCFG_ALL_SECURITY_ENABLED}}
+  exporter:
+    count: 1
+    {{#ENABLE_VIRTUAL_NETWORK}}
+      networks:
+        {{VIRTUAL_NETWORK_NAME}}:
+          labels: {{VIRTUAL_NETWORK_PLUGIN_LABELS}}
+    {{/ENABLE_VIRTUAL_NETWORK}}
+    uris:
+      - {{ELASTICSEARCH_JAVA_URI}}
+      - {{PROMETHEUS_EXPORTER_URI}}
+    tasks:
+      node:
+        goal: RUNNING
+        cpus: {{PROMETHEUS_EXPORTER_CPUS}}
+        memory: {{PROMETHEUS_EXPORTER_MEM}}
+        labels: "DCOS_METRICS_FORMAT:prometheus,DCOS_METRICS_PORT_INDEX:0,DCOS_METRICS_ENDPOINT:/metrics"
+        ports:
+          exporter:
+            port: 9114
+        volume:
+          path: "container-path"
+          type: {{PROMETHEUS_EXPORTER_DISK_TYPE}}
+          size: {{PROMETHEUS_EXPORTER_DISK}}
+          {{#PROMETHEUS_EXPORTER_VOLUME_PROFILE}}
+            profiles:
+              - {{PROMETHEUS_EXPORTER_VOLUME_PROFILE}}
+          {{/PROMETHEUS_EXPORTER_VOLUME_PROFILE}}
+        cmd: |
+          set -ex
+          export JAVA_HOME=$(ls -d $MESOS_SANDBOX/jdk*/)
+          exec ./elasticsearch_exporter-1.1.0.linux-amd64/elasticsearch_exporter \
+            --es.uri="http://master.${FRAMEWORK_NAME//\//}.l4lb.thisdcos.directory:9200" \
```

Comment:
We'll need to make this work when the Elasticsearch service is deployed with transport encryption too. We could probably rename this environment variable to be more general and use it here too instead of the hardcoded `http` protocol: https://github.com/mesosphere/dcos-elastic-service/blob/bbf75b8266f24a42e4f524d426f83f53c56b061e/frameworks/elastic/universe/marathon.json.mustache#L65